app/docs/enterprise/0.32-x/getting-started/adding-consumers.md

```diff
@@ -85,11 +84,9 @@ Plugins, feel free to read more on Kong in one of the following documents:
 - [Admin API Reference][API]
 - [Clustering Reference][cluster]
 
-
 [key-auth]: /plugins/key-authentication
 [API-consumers]: /docs/latest/admin-api#create-consumer
```

Comment:
I think, but am not certain, that these should refer to a matching version of the docs, not /latest/

Reply:
This isn't possible due to the way the templates currently work. CE docs use `/docs/{{page.kong_version}}/admin-api#create-consumer`, but https://getkong.org/docs/0.32.x/admin-api/#create-consumer does not exist--that particular doc only exists for CE.

I'd very much like something that would allow for links like `/docs/{{page.kong_core_version}}/whatever` that could be used in both CE and EE docs (it'd evaluate to either the CE version for a CE doc or the base CE version for an EE doc), but that requires Jekyll work that's well outside the scope of this PR.

Reply:
The "matching version" version of link would be `/docs/0.13.x/admin-api/#consumer-object` because Kong CE version 0.13 "matches" Kong EE version 0.32. 

Sorry if my "matching version" terminology isn't correct here - I'd be happy to know what we call this :)

Reply:
@rainest will you be making the requested changes to this PR?